playbooks/common/openshift-cluster/upgrades/v3_5/validator.yml

```diff
@@ -0,0 +1,11 @@
+---
+###############################################################################
+# Post upgrade - Upgrade job storage
```

Comment:
This file seems to be called with the tag `pre_upgrade` in the above file (`upgrades/v3_5/upgrade_control_plane.yml`). However the comment header in this says it is for **Post upgrade - Upgrade job storage**.

But.... all it does is include `lib_openshift` and then *Check for invalid namespaces and SDN errors*.

This does not all match up. This **is** a pre-upgrade step, is it just mis-labeled?